libs/tracing/src/main/kotlin/net/corda/tracing/impl/BraveTracingService.kt

```diff
@@ -70,20 +70,26 @@ internal class BraveTracingService(serviceName: String, zipkinHost: String, samp
                     .build()
             )
 
-        // The console reporter is useful when debugging test runs on the combined worker.
-        // uncomment it to enable it.
-        val reporters = mutableListOf<Reporter<Span>>(/*Reporter.CONSOLE*/)
-        val reporter = CombinedSpanReporter(reporters)
+        //Establish zipkin connection iff url host is provided
+        if (zipkinHost != null){
 
-        val zipkinUrl = "$zipkinHost/api/v2/spans"
-        val spanAsyncReporter =
-            AsyncReporter.create(URLConnectionSender.create(zipkinUrl))
-                .also(resourcesToClose::push)
-        reporters.add(spanAsyncReporter)
+            // The console reporter is useful when debugging test runs on the combined worker.
+            // uncomment it to enable it.
+            val reporters = mutableListOf<Reporter<Span>>(Reporter.CONSOLE)
```

Comment:
`Reporter.CONSOLE` should still be commented out

```suggestion
            val reporters = mutableListOf<Reporter<Span>>(/*Reporter.CONSOLE*/)
```